examples/advanced/quantum_volume_test.py

```diff
@@ -7,6 +7,7 @@
 import cirq
 
 
+@cirq.testing.skip_if_module_not_exists(module="cirq.google")
```

Comment:
Is this skip still needed? It looks like the `cirq.google` dependencies have been removed.

Reply:
yeah, the example is using cirq.google.Bristlecone

Reply:
I think the `examples` package is fine to have this skipping, as it is are a cross-module package